README.Rmd

```diff
@@ -98,7 +103,3 @@ Remember:
 Phew, that was easier than I thought. Last but not least of all:
 
 Have fun creating and using this extension and [Sonatype OSS Index](https://ossindex.sonatype.org/), we are glad to have you here!
-
-## Getting help
-
-The best way to get in touch is to join us on the [oysteR Gitter](https://gitter.im/sonatype-nexus-community/oysteR) - see you there!
```

Comment:
Is this no longer true? I did see some Gitter activity earlier this month

Reply:
Sorry, I've never heard of anyone using gitter in R. I kept the icon at the top. 

Happy to add back in.

Reply:
Well, as project a owner, if you're not using the Gitter then perhaps we are giving bad info as to "the best way to get in touch" :-)

I think specifying some way to reach out to talk to a human is helpful. Maybe we can keep this section, but prompt folks to open an issue if they want to discuss something about the project? Or we could list both the Gitter channel and GH Issues as potential communication channels for folks stumbling on this project who want to reach out. Or perhaps there are some different, more conventional channels in the R community that we could use instead?

Reply:
Added the gitter section back with a sentence on issues. Thanks